boards/common/nrf52xxxdk/board.c

```diff
@@ -21,8 +21,46 @@
 #include "cpu.h"
 #include "board.h"
 
+#if defined(BOARD_NRF52DK)
```

Comment:
Why not but the macros in the board.h file?

Reply:
Or periph_conf.h ?

Reply:
I think this should be in place that LED0 macros are defined since it is basically a friendly name for  a pin.

Reply:
But it is just for consideration, not trying to push.

Reply:
> Hmm, have spend some more thought on this. Since the config is held in persistent memory, there is not need to check on every system boot.
>
> I will instead provide stand-alone application which provides the means to program the reset pin.

just wait some more minutes :-)